pom.xml

```diff
@@ -19,143 +19,185 @@
   under the License.
```

Comment:
Eclipse is still fighting me on it.  It looks correct in my local editor and not in the commit.  I'll fix it shortly.